CMakeLists.txt

```diff
@@ -445,6 +445,8 @@ IF (WIN32)
   SET (DEFAULT_PLUGIN_SUBDIR  plugins)
   SET (DEFAULT_INCLUDE_SUBDIR include)
 
+  SET (DEFAULT_SERVER_MODULE_SUBDIR server)
```

Comment:
Plugins path is strongly related to plugins for qgis core (loaders....),  I would prefer not to interfere with that atm. In fact, it is actually a quick and easy way to build module's path from QgsApplication::libexecPath().  
Anyway, nothing is fixed now and we can change that later.